.all-contributorsrc

```diff
@@ -140,6 +140,15 @@
       "contributions": [
         "review"
       ]
+    },
+    {
+      "login": "jackyho112",
+      "name": "jackyho112",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/25375401?v=3",
+      "profile": "https://github.com/jackyho112",
+      "contributions": [
+        "code"
```

Comment:
You're selling yourself short :wink: you also contributed tests! If you would like, please add `"test"` to this array then update the contributor table as documented in the CONTRIBUTING.md 😄